src/sys/socket/ffi.rs

```diff
@@ -20,7 +20,10 @@ pub type type_of_cmsg_len = socklen_t;
 #[cfg(target_os = "macos")]
 pub type type_of_cmsg_data = c_uint;
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(target_os = "dragonfly")]
```

Comment:
I think this should be `#[cfg(any(target_os = "dragonfly", target_os = "macos"))]`